src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -171,7 +171,7 @@ const CpsOnwardJourney = ({
     : { as: 'section', role: 'region', 'aria-labelledby': labelId };
 
   const Wrapper = isMediaContent
-    ? styled(ConstrainedLargeGrid)`` // Necessary for styling
+    ? ConstrainedLargeGrid
```

Comment:
We might still need the `styled(ConstrainedLargeGrid)` here? Presumably it's so that the the `<Grid>` within it receives styles?